ledger/complete/ledger.go

```diff
@@ -433,3 +441,30 @@ func (l *Ledger) keepOnlyOneTrie(state ledger.State) error {
 	}
 	return nil
 }
+
+func runReport(r ledger.Reporter, p []ledger.Payload, l zerolog.Logger) (ledger.State, error) {
+	l.Info().
+		Str("name", r.Name()).
+		Msg("starting reporter")
+
+	start := time.Now()
+	err := r.Report(p)
+	elapsed := time.Since(start)
+
+	l.Info().
+		Str("timeTaken", elapsed.String()).
+		Str("name", r.Name()).
+		Msg("reporter done")
+	if err != nil {
+		return ledger.State(hash.DummyHash),
+			fmt.Errorf("error running reporter (%s): %w", r.Name(), err)
+	}
+	return ledger.State(hash.DummyHash), nil
```

Comment:
this seems to always return dummyHash?

Reply:
I will remove this so that it only returns an error or nil from the function. There is no reason that a hash needs to be returned.